ui/app/templates/components/alert-popup.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="message {{type.class}}">
+  <div class="columns is-mobile is-variable is-1">
+    <div class="column is-narrow message-icon">
+      <ICon @glyph="{{type.glyph}}" @size="20" @excludeIconClass={{true}} />
+    </div>
+    <div class="column">
+      <button type="button" class="close-button" {{action close}}>
+        <ICon @excludeIconClass=true @glyph="close" @aria-label="Close" />
```

Comment:
```suggestion
        <ICon @excludeIconClass={{true}} @glyph="close" @aria-label="Close" />
```

Bools have to be in curlies (more important with `false`, but good to be consistent).